helliio/01-FFI_Android
# Sprint5c

Sprint 5 Client:

UI
- show coworkers latest location on map

functionality:
- new upload photo system for photo upload: 
- photo upload uses its own upload functions to ensure the upload get done
- upload one photo at a time, until there is no network or no un-uploaded photos.
- new photo download system: 
- download latest photo report list(all informations about a photo report: location, time, description ...) 
- save the list into local database, when it is done, 
- download each photo file from server.
-  each of the download photo file are saved private to the application
- only this application can use the photos downloaded from server, but the photo taking by user are saved publicly as the copyright belongs to the user).
- it works! all major functions works :)
